doc/releasenotes/16_0_0_summary.md

```diff
@@ -34,6 +34,13 @@ Other aspects of the VReplication copy-phase logic are preserved:
 
  Other phases, catchup, fast-forward, and replicating/"running", are unchanged.
 
+### Important bug fixes
+
+#### Corrupted results for non-full-group-by queries with JOINs
+
+An issue in versions `<= v14.0.3` and `<= v15.0.0` that generated corrupted results for non-full-group-by queries with a JOIN
```

Comment:
Why do we need to specify both <= 14.0.3 and <= 15.0.0?

Reply:
To be explicit with where this issue resides. Might help to know that someone running `v15.0.1` looking to upgrade to `v16.0.0` is not affected by that issue and patch.

Reply:
Is it also an issue on 13.0? Or just 14 & 15?

Reply:
It is an issue only in 14 and 15. We were not supporting this kind of query at all before 14.